guides/V1_ActionsEvents.md

```diff
@@ -0,0 +1,198 @@
+---
+copyright: 'Copyright IBM Corp. 2017'
+link: 'actions-and-events'
+is: 'experimental'
+---
+## Actions and Events
+
+Watson Work Services gives your App two patterns for dealing with events in the system and allowing for actions from a 
+client. Your App can Listen to Events, through secure web hooks. Also, your App can handle actions that may be invoked 
```

Comment:
"events in the system" take place either due to changes in the data model and/or due to actions humans take using a participating client in Watson Work Services such as the Workspace clients.

Reply:
ok, I made a change let me know what you think...is this the only change you're requesting?